docs/download_files.md

```diff
@@ -14,6 +14,10 @@ The files associated with each library are (example shown for a library with ID
 
 Every download also includes a single `single_cell_metadata.tsv` file containing metadata for all libraries included in the download.
 
+If a sample includes a library that was processed using spatial transcriptomics, an additional folder, `SCPCL000000_spatial`, will be included inside the sample folder. 
+This folder will contain all output files related to processing the spatial transcriptomics library. 
+See the [description of the Spatial transcriptomics output section below](#download-contents-for-samples-with-spatial-transcriptomics) for more information. 
```

Comment:
I like having this section here, but I wanted to try to make it a bit shorter. I also updated the link to match the change I made to the header (but you may need to update it again if anything changes there!). I still don't love it, but it may just be an awkward concept to get across succinctly.

```suggestion
If a sample includes a library that was processed using spatial transcriptomics, there will be an additional folder in the sample directory, `SCPCL000000_spatial`. 
For more information about the contents of this folder, see the [spatial transcriptomics libraries section below](#spatial-transcriptomics-libraries). 
```